app/public/locales/en/common.json

```diff
@@ -1,10 +1,39 @@
 {
+  "Eligibility": {
+    "baby": "I've had a baby in the past 12 months",
+    "categorical": "2. Please select all the following that apply to your family:",
```

Comment:
Nit: I think it would actually be better to pull out the numbers from this content and see if we can use either an `ol` or use CSS to enumerate.